src/pages/using-spark/components/input.mdx

```diff
@@ -92,6 +93,35 @@ than one option from a list.
   hasHTML
 />
 
+### Huge Checkbox
+[Back to Variants](#variants)
+
+The Huge Checkbox component is intended for flows
+that present single questions in a step by step process
+For this reason, Huge Checkbox should only be
+used for stand-alone question flows.
+
+#### Guidelines
+- Ensure list is in logical order for inputs (groups will stack).
+- Options are short and concise.
+- Recommend content to not exceed 40 characters.
+- The inputs become stacked in a grid. Stacking will
+take place automatically based on options presented.
```

Comment:
```suggestion
take place automatically if you use the layout code based on options presented.
```

We need to make sure they know the automatically part only happens if they use the stack code.